<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReptateMainWindow</class>
 <widget class="QMainWindow" name="ReptateMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RepTate</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MainWindow.qrc">
    <normaloff>:/Icons/Images/Reptate64.ico</normaloff>:/Icons/Images/Reptate64.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="ApplicationtabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="ConsoledockWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>Python/RepTate Console</string>
   </property>
   <property name="statusTip">
    <string>Python/RepTate Console</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTest"/>
   <addaction name="actionReact"/>
   <addaction name="actionMWD"/>
   <addaction name="actionTTS"/>
   <addaction name="actionLVE"/>
   <addaction name="actionNLVE"/>
   <addaction name="actionGt"/>
   <addaction name="actionCreep"/>
   <addaction name="actionSANS"/>
   <addaction name="separator"/>
   <addaction name="actionMaterials_Database"/>
   <addaction name="actionPocket_Calculator"/>
   <addaction name="actionHeader_Converter"/>
   <addaction name="separator"/>
   <addaction name="actionProject"/>
   <addaction name="actionConsole"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
   <addaction name="actionAbout_Qt"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QDockWidget" name="ProjectdockWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <property name="windowTitle">
    <string>Project Explorer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="Projecttree">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 200);</string>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>1</number>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>App</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>NOT COMPLETE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TTS</string>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Dataset1</string>
         </property>
         <item>
          <property name="text">
           <string>File1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>File2</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>Theory1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Theory2</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>LVE</string>
        </property>
        <item>
         <property name="text">
          <string>Dataset1</string>
         </property>
         <item>
          <property name="text">
           <string>File1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>File2</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>Theory1</string>
         </property>
        </item>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/Exit.ico</normaloff>:/Icons/Images/Exit.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Exit Reptate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionReact">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/react.ico</normaloff>:/Icons/Images/react.ico</iconset>
   </property>
   <property name="text">
    <string>React</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionTTS">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/TTS.ico</normaloff>:/Icons/Images/TTS.ico</iconset>
   </property>
   <property name="text">
    <string>TTS</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionLVE">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/LVE.ico</normaloff>:/Icons/Images/LVE.ico</iconset>
   </property>
   <property name="text">
    <string>LVE</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionNLVE">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/NLVE.ico</normaloff>:/Icons/Images/NLVE.ico</iconset>
   </property>
   <property name="text">
    <string>NLVE</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
  <action name="actionGt">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/Gt.ico</normaloff>:/Icons/Images/Gt.ico</iconset>
   </property>
   <property name="text">
    <string>Gt</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+6</string>
   </property>
  </action>
  <action name="actionMaterials_Database">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/Library.ico</normaloff>:/Icons/Images/Library.ico</iconset>
   </property>
   <property name="text">
    <string>Materials Database</string>
   </property>
  </action>
  <action name="actionPocket_Calculator">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/Calculator.ico</normaloff>:/Icons/Images/Calculator.ico</iconset>
   </property>
   <property name="text">
    <string>Pocket Calculator</string>
   </property>
  </action>
  <action name="actionHeader_Converter">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/MatDB.ico</normaloff>:/Icons/Images/MatDB.ico</iconset>
   </property>
   <property name="text">
    <string>Header Converter</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/Reptate.ico</normaloff>:/Icons/Images/Reptate.ico</iconset>
   </property>
   <property name="text">
    <string> About Reptate</string>
   </property>
  </action>
  <action name="actionMWD">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/Mw.ico</normaloff>:/Icons/Images/Mw.ico</iconset>
   </property>
   <property name="text">
    <string>MWD</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionCreep">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/Creep.ico</normaloff>:/Icons/Images/Creep.ico</iconset>
   </property>
   <property name="text">
    <string>Creep</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+7</string>
   </property>
  </action>
  <action name="actionSANS">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/SANS.ico</normaloff>:/Icons/Images/SANS.ico</iconset>
   </property>
   <property name="text">
    <string>SANS</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+8</string>
   </property>
  </action>
  <action name="actionProject">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/Reptate.ico</normaloff>:/Icons/Images/Reptate.ico</iconset>
   </property>
   <property name="text">
    <string>Project</string>
   </property>
   <property name="toolTip">
    <string>Show/Hide Project Explorer</string>
   </property>
   <property name="statusTip">
    <string>Show/Hide Project Explorer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/console.ico</normaloff>:/Icons/Images/console.ico</iconset>
   </property>
   <property name="text">
    <string>Console</string>
   </property>
   <property name="toolTip">
    <string>Show/Hide Console</string>
   </property>
   <property name="statusTip">
    <string>Show/Hide Console</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F5</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/Test.ico</normaloff>:/Icons/Images/Test.ico</iconset>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/qt.ico</normaloff>:/Icons/Images/qt.ico</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/Help.ico</normaloff>:/Icons/Images/Help.ico</iconset>
   </property>
   <property name="text">
    <string>Contents</string>
   </property>
   <property name="toolTip">
    <string>Main Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>ReptateMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
